src/main/java/org/primeframework/mvc/security/cookiesession/InMemorySessionContainer.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2024, Inversoft Inc., All Rights Reserved
+ */
+package org.primeframework.mvc.security.cookiesession;
+
+import java.time.ZonedDateTime;
+
+public class InMemorySessionContainer {
```

Comment:
May be easier to talk this one through - not sure what the purpose of shipping an in memory session container is?

Reply:
Renamed this and clarified but we can discuss